src/main/java/com/simibubi/create/content/kinetics/chainConveyor/ChainConveyorRenderer.java

```diff
@@ -167,38 +270,103 @@ private void renderChains(ChainConveyorBlockEntity be, PoseStack ms, MultiBuffer
 					.overlay(overlay)
 					.renderInto(ms, buffer.getBuffer(RenderType.cutoutMipped()));
 			}
+			if (frustum == null || frustum.testLineSegment((float) (start.x - camPos.x), (float) (start.y - camPos.y), (float) (start.z - camPos.z),
+				(float) (end.x - camPos.x), (float) (end.y - camPos.y), (float) (end.z - camPos.z))) {
+				double pitch = (float) Mth.RAD_TO_DEG * Mth.atan2(diff.y, diff.multiply(1, 0, 1)
+					.length());
+
+				int light1 = LightTexture.pack(level.getBrightness(LightLayer.BLOCK, tilePos),
+					level.getBrightness(LightLayer.SKY, tilePos));
+				int light2 = LightTexture.pack(level.getBrightness(LightLayer.BLOCK, targetPos),
+					level.getBrightness(LightLayer.SKY, targetPos));
+
+				Vec3 startOffset = start.subtract(Vec3.atCenterOf(tilePos));
+
+				ms.pushPose();
+				var chain = TransformStack.of(ms);
+				chain.center();
+				chain.translate(startOffset);
+				chain.rotateYDegrees((float) yaw);
+				chain.rotateXDegrees(90 - (float) pitch);
+				chain.rotateYDegrees(45);
+				chain.translate(0, 8 / 16f, 0);
+				chain.uncenter();
+
+				if (frustum != null) {
+					renderChainWithLod(ms, buffer, animation, light1, light2, camPos, start, end, chain);
+				} else {
+					renderChain(ms, buffer, animation, 0, stats.chainLength(), light1, light2, false);
+				}
+				ms.popPose();
+			}
+
+			if (frustum == null)
+				continue;
+
+			float renderDistance = Minecraft.getInstance().gameRenderer.getRenderDistance();
+			if (camPos.distanceToSqr(targetPos.getCenter()) <= renderDistance * renderDistance)
```

Comment:
Same thing here, if the target is outside render distance but this block is in, shouldn't we still try to render the chain?